services/provider/server/consumer.go

```diff
@@ -27,6 +28,12 @@ type ocsConsumerManager struct {
 	mutex        sync.RWMutex
 }
 
+type statusPatch struct {
+	Op    string      `json:"op"`
+	Path  string      `json:"path"`
+	Value interface{} `json:"value"`
+}
+
```

Comment:
- thought about it, however unsure the compiler move local types to global region or not and so defined at pkg level rather than redefining during every func call

Reply:
moved to function as anon struct.

Reply:
removed the struct